site/en/docs/lighthouse/index.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: 'layouts/project-landing.njk'
+title: 'Lighthouse'
+description: 'Lighthouse is an open-source, automated tool for improving the quality of your web apps.'
```

Comment:
FYI: This first pass doesn't include translations for the title or the description.